test/fixtures/completions/exportedMemberDocs.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+)
+
+//
+var Language = "english" // should not invoke completion since this is line comment
+
+var (
+	//
+	Greeting = "Hello"
+	//
+	GreetingText = "Hello World!"
+)
+
+const (
+	//
+	HelloStatus = 0
+	//
+	GreetingStatus = 1
+)
+
+//
```

Comment:
Keep the cursor between the two `/` and manually trigger completions by pressing `Ctrl+Space`. We get the completion for `SayHello`, but we shouldn't

Reply:
Have added logic to handle this case.